draft-smith-rats-evidence-trans.md

```diff
@@ -249,12 +249,21 @@ The binary representation of DTI.`type` MUST be equivalent to the binary represe
 > > If _m_.`notTcb` = 1 AND _f_.`notTcb` = 0; **set**(ECT.`element-list`.`element-map`.`measurement-values-map`.`flags`.`is-tcb` = TRUE).
 
 {: dtt-enum}
-* The signer of the certificate containing DTI is copied to the ECT.`authority` field.
-The signer identity MUST be expressed using `$crypto-key-type-choice`.
-A profile or other arrangement is used to coordinate which `$crypto-key-type-choice` is used for both Evidence and Reference Values.
+* The ECT.`authority` field is set up based on the signer of the certificate containing DTI as described in {{sec-authority}}.
 
 The completed ECT is added to the `ae` list.
 
+## Authority field in DICE/SPDM ECTs {#sec-authority}
+
+The ECT authority field is an array of `$crypto-keys-type-choice`s.
+
+When adding Evidence to the ACS, the Verifier SHALL add the public key representing the signer of the element (for example the DICE certificate or SPDM MEASUREMENTS response) to this array.
+The Verifier SHALL also add the signer of each certificate which has authorized (either directly or indirectly) the signer of the element.
+
+Having each authority in a certificate path in the ECT `authority` field lets conditional endorsement conditions match multiple authorities or match an authority that is scoped more broadly than the immediate signer of the Evidence artifact.
+
+Each signer authority MUST be expressed using the COSE_Key varient of `$crypto-key-type-choice`.
```

Comment:
```suggestion
Each signer authority value MUST be represented in a format that can be translated to `tagged-cose-key-type`.
```
This wording allows for CoRIM to document key translation requirements while not overstepping into CoRIM normative.